overrides/groovy/postInit/Post-Initial/Main/Mod-Specific/jei.groovy

```diff
@@ -4,6 +4,64 @@ import com.nomiceu.nomilabs.util.LabsModeHelper
 
 // AR
 mods.hei.removeAndHide(item('advancedrocketry:crystal:*')) // Random Crystal Blocks
+mods.hei.removeAndHide(item('densemetals:dense_platinum_ore')) //Dense Platinum ore
+mods.hei.removeAndHide(item('gregtech:meta_crushed_centrifuged:81')) // Centrifuged Ore
+mods.hei.removeAndHide(item('gregtech:ore_trona_0', 0)) //trona ore
+mods.hei.removeAndHide(item('gregtech:ore_trona_0', 1)) //trona ore
+mods.hei.removeAndHide(item('gregtech:ore_trona_0', 2)) //trona ore
+mods.hei.removeAndHide(item('gregtech:meta_dust_small:2031')) // small dust
+mods.hei.removeAndHide(item('densemetals:dense_diamond_ore')) //Dense Diamond ore
+mods.hei.removeAndHide(item('densemetals:dense_quartz_ore')) // dense quartz ore
+mods.hei.removeAndHide(item('densemetals:dense_copper_ore')) // dense copper ore
+mods.hei.removeAndHide(item('nomilabs:meta_crushed_purified', 110)) // purified crushed ore
+mods.hei.removeAndHide(item('nomilabs:meta_dust_pure:110')) // pure dust
+mods.hei.removeAndHide(item('densemetals:dense_thorium_ore')) // dense thorium ore
+mods.hei.removeAndHide(item('gregtech:meta_crushed_purified:2031')) // purified crushed ore
+mods.hei.removeAndHide(item('nomilabs:meta_dust_small:110')) // small dust
+mods.hei.removeAndHide(item('gregtech:meta_crushed:2031;')) // crushed ore
+mods.hei.removeAndHide(item('densemetals:dense_emerald_ore')) // dense emerald ore
+mods.hei.removeAndHide(item('densemetals:dense_plutonium_ore')) // dense plutonium ore
+mods.hei.removeAndHide(item('gregtech:meta_dust_tiny:2031')) // tiny dust
+mods.hei.removeAndHide(item('nomilabs:meta_dust_impure:110')) // impure dust
+mods.hei.removeAndHide(item('densemetals:dense_lapis_ore')) // dense lapis ore
+mods.hei.removeAndHide(item('densemetals:dense_lead_ore')) // dense lead ore
+mods.hei.removeAndHide(item('gregtech:meta_crushed_centrifuged:2031')) // centrifuged ore
+mods.hei.removeAndHide(item('densemetals:dense_silver_ore')) // dense silver ore
+mods.hei.removeAndHide(item('densemetals:dense_iridium_ore')) // dense iridium ore
+mods.hei.removeAndHide(item('nomilabs:meta_dust_tiny:110')) // tiny dust
```

Comment:
Tiny Dust of what? (cleanup related comments as well, not needed if it is obvious due to the usage of a util function)

For these where misc things are removed, please use the `metaitem` bracket handler, and consider moving it into a util function.